### DISCUSSION TITLE
Multi website with Bundling not working properly

### DISCUSSION THREAD

**Discussion by niravGV** [breezefront/community#127] [1. Help] [answered]
Hello @vovayatsyuk 

We are using the Mobile theme as Breeze theme and we have to enabled the bundling option and other js minify and merge configuration on.

Somehow the bundle created by breeze theme is not loading properly with a locale for E.g:-

website US - breeze bundling and minify working fine.
website AU/CA - other website having bundling and the minify not working fine loading proper locale on the webiste.

Can you please help me if you have idea about it.

Thanks

**Comment by vovayatsyuk** [comment]
What do you mean by "not working fine loading proper locale on the website"?

**Comment by niravGV** [comment]
it mean the CA website some time load the AU locale and AU website load CA locale that kind of problem occured.

**Reply by vovayatsyuk** [reply, marked as answer]
Do you use the latest breeze version?

A similar bug was fixed in https://github.com/breezefront/module-breeze/commit/8abef8d43b0b82b70577f18cbac4aab843573cd2

**Comment by niravGV** [comment]
opps i am not using latest version that code missing in the same block file,
Let me update the breeze module and check it again.

Thanks for your quick help @vovayatsyuk
